lib/game.rb

```diff
@@ -134,18 +140,22 @@ def display_player_results
   def display_computer_results
     if @player_board.cells[@computer_shot].render == "M"
       puts "The computers shot was a Miss"
+      puts "The computer shot coordinate #{@computer_shot}"
     elsif @player_board.cells[@computer_shot].render == "H"
       puts "The computers shot was a Hit"
-    elsif @player_board.cells[@computer_shot].render == "S"
+      puts "The computer shot coordinate #{@computer_shot}"
+    elsif @player_board.cells[@computer_shot].render == "X"
+      (@player_cruiser.sunk? == true || @player_submarine.sunk? == true)
```

Comment:
We tried to address the problem of x's rendering when there had been 3 combined shots on the ships. Its not working, but this makes more sense because it is requiring one of the individual ships to be sunk